models/user.model.js

```diff
@@ -36,7 +36,11 @@ const UserSchema = new Schema({
     },
     secret: String,
     imgUrl: String,
-    phone: String,
+    phoneNumber: String,
+    birthday: String,
+    location: String,
+    bio: String,
+    twitterHandle: String,
```

Comment:
what does this add? like will it be integrated with the twitter thing on your profile?

Reply:
yup got the timeline working!